pyk/src/pyk/kore/rpc.py

```diff
@@ -56,8 +56,50 @@ def __init__(self, message: str, code: int, data: Any = None):
 
 
 class Transport(ContextManager['Transport'], ABC):
+    _bug_report: BugReport | None
+    _bug_report_id: str | None
+
+    def __init__(self, bug_report_id: str | None = None, bug_report: BugReport | None = None) -> None:
+        self._bug_report_id = bug_report_id
+        self._bug_report = bug_report
```

Comment:
```suggestion
    def __init__(self, bug_report: BugReport | None = None, bug_report_id: str | None = None) -> None:
        self._bug_report = bug_report
        self._bug_report_id = bug_report_id
```

Reply:
Is it valid to have a `bug_report_id` but not a `bug_report`? If it isn't, `ValueError` should be raised when it happens.

Reply:
Added this error.